reactioncommerce/reaction
# fix: orders performance

Resolves #5094  
Impact: **major**  
Type: **performance**

## Issue

The orders admin panel is performing very poorly.

## Solution

- Remove aggregations in an attempt to improve performance
- Rebuild the Orders table to be more performant

## Breaking changes

Multi-shop orders will no longer work

## Testing
1. Make orders, many orders, or use a data-set that provides many orders
2. Paginate through the pages and ensure they still paginate
3. Fulfill an order to completion
4. Cancel an order
5. View PDF preview, and ensure it has all the correct data
6. (basically make sure all parts of order fulfillment still works for an order with a single shop
